oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/VocabularyTreeSelectField/TreeSelectFieldModal.jsx

```diff
@@ -285,62 +286,72 @@ export const TreeSelectFieldModal = ({
     }
   };
 
+  useEffect(() => {
```

Comment:
Can we do it without resorting to useEffect? When in multiselect mode, this effect is completely unnecessarily registered & evaluated everytime someone selects something...